src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -622,6 +622,10 @@ bool primitive_inst::update_impl() {
 }
 
 void primitive_inst::do_runtime_in_place_concat() {
+    GPU_DEBUG_GET_INSTANCE(debug_config);
+    GPU_DEBUG_IF(debug_config->disable_async_compilation) {
```

Comment:
Isn't this what you originally intended?
```suggestion
    GPU_DEBUG_IF(debug_config->disable_runtime_buffer_fusing) {
```

Reply:
oops thank you :D

Reply:
Fixed